test/unit/scripting_spec.js

```diff
@@ -159,6 +159,64 @@ describe("Scripting", function () {
         done.fail(ex);
       }
     });
+
+    it("should get field using a path", async function (done) {
```

Comment:
Why async and done?

Reply:
Yeah it's useless.
Let's fixing that in a follow-up.

Reply:
Is there a reason this needs to be done in a follow up? Seems like a good time to fix it here.

Reply:
Done here: https://github.com/mozilla/pdf.js/pull/12830/commits/ccef734ebb80e50b0ed15ddc0f4c26cc1619f1ad